djangocms_form_builder/actions.py

```diff
@@ -119,7 +119,7 @@ def execute(self, form, request):
             keys = {}
             defaults = {
                 "form_name": get_option(form, "form_name"),
-                "form_user": request.user,
+                "form_user": request.user if get_option(form, "login_required", False) else None
```

Comment:
```suggestion
                "form_user": None if request.user.is_anonymous else request.user
```
How about still recording if the form does not require login?